src/gameplay/types.ts

```diff
@@ -25,3 +25,20 @@ export interface Role {
 export interface See {
   spies: string[];
 }
+
+type Team = 'Resistance' | 'Spy';
+
+export interface Match {
+  winningTeam: Team;
+  playerTeam: Team;
+  opponentTeamRating: number;
```

Comment:
What's player and opponent? I feel like these variables in Match aren't what I'd expect?